src/dwfl_symbol.cc

```diff
@@ -70,6 +69,22 @@ bool symbol_get_from_dwfl(Dwfl_Module *mod, ProcessAddress_t process_pc,
       symbol._srcpath = std::string(localsrcpath);
       symbol._lineno = static_cast<uint32_t>(linep);
     }
+    Offset_t dw_bias;
+    Dwarf_Line *dwarf_line = dwfl_dwarf_line (line, &dw_bias);
+    Dwarf *dwarf = dwfl_module_getdwarf(mod, &dw_bias);
+    if (dwarf && dwarf_line) {
+      const char *inline_func_name = dwarf_linefunctionname(dwarf, dwarf_line);
+      if (inline_func_name) {
```

Comment:
@sanchda you were asking about inlining.
This is very hacky, but it allows us to get the info